python/sparknlp/comet_logger.py

```diff
@@ -0,0 +1,221 @@
+import comet_ml
+
+import threading
+import time
+import os
+
+
+class CometLogger:
+    def __init__(
+        self,
+        workspace=None,
+        project_name=None,
+        comet_mode=None,
+        experiment_id=None,
+        tags=None,
+        **experiment_kwargs,
+    ):
+        self.experiment = comet_ml.Experiment()
+        self.comet_mode = comet_mode
+        self.workspace = workspace
+        self.project_name = project_name
+        self.experiment_id = experiment_id
+        self.experiment_kwargs = experiment_kwargs
+
+        self.experiment = self._get_experiment(
+            self.comet_mode,
+            self.workspace,
+            self.project_name,
+            self.experiment_id,
+            **self.experiment_kwargs,
+        )
+        self.experiment.log_other("Created from", "SparkNLP")
+        if tags is not None:
+            self.experiment.add_tags(tags)
+
+    def _get_experiment(
```

Comment:
Since we're monitoring the log file to log Epoch/Steps, it's a bit tricky to continue an experiment. I'm wondering if we should just limit it to Online/Offline Experiments and leave Existing Experiments out of the integration.